app/src/main/res/navigation/onboarding_graph.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sometimes having onboarding and auth flows in the same graph is the right choice.

However, in a future exercise we want to add a way to skip onboarding if the user has seen it already. Also we'll implement forgot/reset password, which are unrelated to onboarding but do relate to authentication.

I suggest for this application that you have a graph for onboarding and one for authentication